index.html

```diff
@@ -394,7 +395,18 @@ <h3>
         <p>A <a>PerformanceNavigationTiming</a> has an associated
         {{NotRestoredReasons}} <a data-dfn-for="PerformanceNavigationTiming"><dfn>not restored reasons</dfn></a>.
 
-        <p>A {{PerformanceNavigationTiming}} has an associated null or [=service worker timing info=] 
+        <p>A <a>PerformanceNavigationTiming</a> has an associated
+        {{PerformanceTimingConfidence}} <a data-dfn-for="PerformanceNavigationTiming"><dfn>confidence value</dfn></a>.
+
+        <p>A <a>PerformanceNavigationTiming</a> has an associated real number
+        <a data-dfn-for="PerformanceNavigationTiming"><dfn>randomized trigger rate</dfn></a>
+        which is implementation-defined.</p>
```

Comment:
Can you link "implementation defined" here and elsewhere? `<a>implementation-defined</a>`  or `[=implementation-defined=]` will probably do.